go.mod

```diff
@@ -1,52 +1,53 @@
 module github.com/choria-io/asyncjobs
 
-go 1.21
+go 1.24
```

Comment:
this should be `go 1.23.6` and if you dont mind doing the CI stuff that'd be great also, test against 23 and 24